### PR TITLE
Experiments template

### PR DESCRIPTION
**Pull request by t-kimber**
In order to have the ligand-based models running, the groups have to be imported per kinase, e.g. `O60674__pIC50Measurement.parquet` and not `pIC50Measurement.parquet` as in the kinase-informed setting.

# Important
1 . This changes the notebook template, which affects the model in the kinase-informed setting.
2. The notebooks still don't run until the end because of lack of data points (we are using a subsample of chembl of 1k points): some metrics cannot be computed and therefore the notebook fails.

### CODE REVIEW

File: experiments/torch-train-test-debug-template.ipynb [hunk 1]
```diff
@@ -345,7 +345,7 @@
     "    train_loss = 0.0\n",
     "    val_loss = 0.0\n",
     "    for key, loader in tqdm(dataloaders.items(), desc=\"Datasets\", leave=False):\n",
-    "        mtype_class = import_object(f\"kinoml.core.measurements.{key}\")\n",
```
Comment (schallerdavid) [inline experiments/torch-train-test-debug-template.ipynb:348]: Add an if statement to switch between kinase specific models and kinome wide models.

Comment (t-kimber) [reply experiments/torch-train-test-debug-template.ipynb:348]: @schallerdavid , this workaround does the trick! The PR is ready to be merged!

**Comment by schallerdavid** [comment]
Hey Talia, also seeing a merge conflict here.

**Comment by t-kimber** [comment]
@schallerdavid , this time it's ready! Feel free to merge if you approve :)

**Review by schallerdavid — APPROVED** [review]

**Pull request merged**